applications/CompressiblePotentialFlowApplication/python_scripts/apply_far_field_process.py

```diff
@@ -22,6 +22,7 @@ def __init__(self, Model, settings ):
         default_parameters = KratosMultiphysics.Parameters( """
```

Comment:
```suggestion
        if not settings.Has("angle_of_attack_units"):
            KratosMultiphysics.Logger.PrintWarning("ApplyFarFieldProcess", "'angle of attack_units' is not provided. Using 'radians' as default angle of attack unit.")

        default_parameters = KratosMultiphysics.Parameters( """
```

Reply:
By doing the check before validating the parameters we can distinguish the cases in which the user specifies the units from those which take the default.